requirements.txt

```diff
@@ -53,6 +56,7 @@ requests==2.18.1
 s3transfer==0.1.10        # via boto3
 simplegeneric==0.8.1      # via ipython
 six==1.10.0               # via cryptography, packaging, prompt-toolkit, python-dateutil, traitlets, zeep
+south==1.0.2
```

Comment:
I think it was added here as a result of django-elastic-transcoder being a dependency, from which south was just removed last week. I'll get rid of it here.